src/eme.js

```diff
@@ -279,11 +278,11 @@ export const standard5July2016 = ({
       });
     }).catch(
       videojs.log.error.bind(videojs.log.error,
-                             'Failed to create and initialize a MediaKeys object')
+                            'Failed to create and initialize a MediaKeys object')
     );
   }
 
-  addSession({
+  const addSessionPromise = Promise.resolve(addSession({
```

Comment:
Any specific reason this needs to be a promise?

Reply:
If it's for testing to see whether the right initData gets used, can you just check `video.pendingSessionData` after the test?